### ISSUE TITLE
Add/Remove VLANs in interface bulk edit

### ISSUE THREAD

**Issue by JulianJacobi** [netbox-community/netbox#4985]
### Environment
* Python version:  3.7
* NetBox version:  2.8.6

### Proposed Functionality
Provide the ability to add/remove tagged and native VLANs at interface bulk edit.

### Use Case
In our case we have many Multi-Chassis-Link-Aggregations, so mostly if we need to add a VLAN to an interface we need to add a VLAN to two or more interfaces in real. If there would be the ability to add/remove VLANs at interface bulk edit, this would be a real time saver.

### Database Changes
None.

### External Dependencies
None.

**Comment by kkthxbye-code** [comment]
Isn't this already present?

**Comment by jeremystretch** [comment]
Yes, it was added some time ago, though I can't find a specific FR for it.
